product_variant_configurator/models/product_product.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+# © 2015 Oihane Crucelaegui - AvanzOSC
+# © 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# © 2016 ACSONE SA/NV
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+
+from openerp import api, exceptions, fields, models, _
+
+
+class ProductProduct(models.Model):
+    _inherit = ['product.product', 'product.configurator']
+    _name = "product.product"
+
+    # This is needed as the AbstractModel removes the delegated related field
+    name = fields.Char(related="product_tmpl_id.name")
```

Comment:
@pedrobaeza Do you remember why the `name` field is redefined in the `product.configurator` model ?

If this is really needed, can it be possible to redefine this field using `compute` instead of `related` ?
The `related` attribute has precedence over `compute`, and it's not possible to empty it in v9 (this is now possible in v10).

Reason : I have a module that redefines the `name` field with `compute`, and don't find any other way to make both modules work together : I'd prefer to keep my custom `compute` behaviour, but even if my module depends on this one, the `related` overrides my `compute` method.